docs/gen/BondCards.html

```diff
@@ -261,7 +261,7 @@ <h3 class="subsection-title">Methods</h3>
             
 <hr>
 <dt>
-    <h4 class="name" id="field"><span class="type-signature"></span>field(selector, type [, compareValue] [, css] [, placeholder] [, successColor] [, errorColor] [, color] [, lineHeight] [, fontSize] [, fontFamily] [, disabled] [, readOnly] [, autoFocus])</h4>
+    <h4 class="name" id="field"><span class="type-signature"></span>field(selector, type [, css] [, placeholder] [, successColor] [, errorColor] [, color] [, lineHeight] [, fontSize] [, fontFamily] [, disabled] [, readOnly] [, autoFocus])</h4>
```

Comment:
Maybe you didn't pull the latest before generating docs? It looks like it removed my compareValue additions. You just might need to make sure you merged master and rerun the doc generation.